### ISSUE TITLE
Communicate inside the PR

### ISSUE THREAD

**Issue by asbjornu** [cirrus-actions/rebase#42]
After `/rebase` is triggered, it would be nice to not have to manually going to "Actions" and click through to the `rebase` workflow to see how it is doing. When the action is triggered, it would be nice to get some sort of feedback in the PR and if it fails, the error message could be posted as a comment in the PR.

**Comment by Paulomart** [comment]
I thought about using reaction for this, but then where is no way to display the error

**Comment by asbjornu** [comment]
@Paulomart, what do you mean "no way to display the error"? Many code review tools, such as [reviewdog](https://github.com/reviewdog/reviewdog) and [Codacy](https://www.codacy.com/) are triggered with GitHub Actions and report their findings through GitHub's API with comments and annotations in the pull request. Is this not possible to do with the `rebase` action?

**Comment by Paulomart** [comment]
Yeah that would be totally possible, but I think it would be a lot of noice.

**Comment by asbjornu** [comment]
I would say that the rebase action builds up to the expectancy of being able to communicate with comments in the PR, as that's how the rebase is triggered to begin with; by typing `/rebase` into a comment in the PR. This sort of human-bot-communication is exactly how [Dependabot](https://dependabot.com/) works. See SwedbankPay/swedbank-pay-sdk-dotnet#44 as an example.

As such, I wouldn't consider a single comment explaining how the rebase can't be performed as noise, I would consider it valuable and expected information. Having to click through to the GitHub Action and dig in the logs to even discover that the rebase can't be performed and why is not the best user experience.

**Comment by asbjornu** [comment]
I would actually love it if the rebase action behaved almost exactly like Dependabot. Here's a more concrete suggestion:

1. When the rebase is started, the action could add a 👍 to the comment containing `/rebase`.
2. A [GitHub Check](https://developer.github.com/v3/checks/) is added to the PR to visualize the status of the rebase: Yellow for "in progress", green for completed successfully and red for completed unsuccessfully.
3. When the rebase can't be performed, a comment explaining why is posted in the PR, with a link to the exact place it failed in the logs.
